Documentation: Update Storage docs - Backup storage volumes [WD-22293] (#15685)
canonical/lxd: doc/howto/storage_backup_volume.md

To retain a specific snapshot even if a general expiry time is set, use the `--no-expiry` flag.
<!-- Include end create snapshot options -->

````
```` {group-tab} UI

To create a snapshot of a custom storage volume, navigate to the overview page of the target volume. Select the {guilabel}`Snapshots` tab, and click {guilabel}`Create snapshot`.

```{figure} /images/storage/storage_volumes_snapshots_tab.png
:width: 80%
:alt: LXD Storage Volumes - Snapshots tab
```

> In the modal that appears, you can provide the snapshot with a name and expiry date and time. When configured, click {guilabel}`Create snapshot` to finish the process.

Reviewer: Removing the line about clicking `Create snapshot` in the spirit of leaving out anything that's super obvious.

    lxc storage volume delete <pool_name> <volume_name>/<snapshot_name>

````
```` {group-tab} UI

> To view, edit or delete a snapshot of a storage volume, navigate to the overview page of the target volume and select the {guilabel}`Snapshots` tab.

Reviewer: ```suggestion
To view, edit or delete a storage volume snapshot, navigate to the {guilabel}`Snapshots` tab for the target volume.
```

    lxc storage volume copy <source_pool_name>/<source_volume_name>/<source_snapshot_name> <target_pool_name>/<target_volume_name>

````
```` {group-tab} UI

To restore a storage volume, navigate to the overview page of the target volume and select the {guilabel}`Snapshots` tab.

> In the list view, hover over a snapshot row to highlight snapshot actions. Click {guilabel}`restore`.

Reviewer: ```suggestion
```

> In the confirmation dialogue that appears, click {guilabel}`Restore snapshot` to finish the process.

Reviewer: ```suggestion
```

: By default, the export file contains all snapshots of the storage volume.
  Add this flag to export the volume without its snapshots.

````
```` {group-tab} UI

> To export a storage volume, navigate to the overview page of the target volume and select {guilabel}`Export` in the top right of the page.

Reviewer: ```suggestion
To export a storage volume, navigate to the {guilabel}`Overview` tab for the target volume and select the {guilabel}`Export` button.
```

> In the resulting modal, configure the export file for the storage volume and click {guilabel}`Export volume` to finish the process.

Reviewer: ```suggestion
In the resulting modal, configure the export file for the storage volume, including its compression mode and expiration.
```

If you do not specify a volume name, the original name of the exported storage volume is used for the new volume.
If a volume with that name already (or still) exists in the specified storage pool, the command returns an error.
In that case, either delete the existing volume before importing the backup or specify a different volume name for the import.

````
```` {group-tab} UI

> To restore a storage volume from an export file, navigate to {guilabel}`Volumes` and select {guilabel}`Create volume` in the top right of the page.

Reviewer: ```suggestion
To restore a storage volume from an export file, select {guilabel}`Volumes` from the main navigation, then click the {guilabel}`Create volume` button.
```

> Click {guilabel}`Upload volume file` in the action bar of the volume creation form. In the resulting modal, upload the volume file and configure the storage pool for the storage volume to be created by the export file and click {guilabel}`Upload and Create` to finish the process.

Reviewer: I'm not seeing the `Upload volume file` option on Version 6.4-ui-0.17.  Does it exist on a different version?

Author: It hasn't been merged yet, but it's on it's last legs :)

```{figure} /images/storage/storage_volumes_snapshots_import.png
:width: 60%
:alt: LXD Storage Volumes - Snapshots tab
```

```{note}

> When within a clustered environment, if a cluster member specific storage pool is selected, you can also configure a target cluster member for the volume.

Reviewer: ```suggestion
Within a clustered environment, if a cluster-member-specific storage pool is selected, you can also configure a target cluster member for the volume.
```
